lib/chain.ts

```diff
@@ -0,0 +1,8 @@
+//TODO - query from chain
+export function getAddressType(_address: string) {
+  if (Math.random() < 0.5) {
+    return "account";
+  }
+
+  return Math.random() < 0.5 ? "contract" : "validator";
+}
```

Comment:
Could we mock it in a different way? That would be consistent between renders. Even a file with the list of contract addresses and validator addresses to hardcode/configure those temporarily would be better for testing and presentation purposes.

Reply:
Good point, maybe checking if the address contains some characters? That way it'd work for any address instead of those on a list

Reply:
I was thinking this way too. Anything predictable would work, I think.